msb/rawlogger/msb_rawlogger.py

```diff
@@ -15,15 +15,16 @@ def signal_handler(sig, frame):
 
 
 class RawLoggerService:
-    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber):
+    def __init__(self, config: RawLoggerConf, subscriber: ZMQRawSubscriber, excluded_topics):
+        self.excluded_topics = excluded_topics
```

Comment:
Consider casting `exluding_topics` to a set (`self.excluded_topics = set(excluded_topics)`).
Sets are optimized for frequent "is in" queries.

Reply:
Please give a more detailed explanation of your suggestion.

Reply:
Set is another basic data structure in python next to dict and list. It is optimized for frequently querying whether an object is in the set. As you "ask" that question very often in the run method in `if topic_decoded not in self.excluded_topics:` it does make sense to use a set here, i.e. save excluded topics as `self.excluded_topics = set(excluded_topics)` (nothing else has to change). 


(A bit off-topic: Sets in python also support those set operations you know from math, like intersection and difference)

Reply:
Incorporated your suggestion.